gravitee-apim-plugin/gravitee-apim-plugin-endpoint/gravitee-apim-plugin-endpoint-mock/src/main/resources/plugin.properties

```diff
@@ -4,3 +4,4 @@ version=${project.version}
 description=${project.description}
 class=io.gravitee.plugin.endpoint.mock.MockEndpointConnectorFactory
 type=endpoint-connector
+features=LIMIT,RESUME
```

Comment:
I would prefer to use lower case here, as defined in the label in the enum

Reply:
👌🏻